pages/accountLists/[accountListId].page.test.tsx

```diff
@@ -111,13 +112,33 @@ describe('AccountListsId page', () => {
   describe('AccountListId Redirects', () => {
     const restApiNotFoundErrorMessage = "Resource 'AccountList' is not valid";
 
+    const makeContext = async (
```

Comment:
I was thinking just having `makeContext` return.

```ts
{
  req: { url: url },
  query: {
    accountListId: 'account-list-1',
  },
} as unknown as GetServerSidePropsContext)
```

I think it's OK and actually good that the tests would explicitly call `getServerSideProps` (i.e. `expect(await getServerServerSideProps(makeContext(...)).toEqual(...)`) since they're testing the functionality of that method.

Reply:
Ok gotcha. I just fixed that